### DISCUSSION TITLE
Pocketbase using custom Certificate Authority to connect to Minio

### DISCUSSION THREAD

**Discussion by svicknesh** [pocketbase/pocketbase#1142] [Q&A]
Discussed in #90 

I have a personal instance of Minio running on my server and would like to connect Pocketbase to it. However, my minio is secured using TLS from a custom certificate authority (that I created). Is there a way to upload or set a path to a set of custom CAs that pocketbase can use to validate connections to a minio (and potentially other services in the future)?

Thank you.

**Comment by ganigeorgiev** [comment]
PocketBase currently doesn't support setting custom certificates pool and the go server will use the globally available from the operating system.
You'll have to register your cert authority on the server where PocketBase is running.
If you are on linux, you can also make use of the env variables `SSL_CERT_FILE` and `SSL_CERT_DIR` to override the system default locations for the SSL certificate file and SSL certificate files directory.

**Reply by ganigeorgiev** [reply]
You can find the default linux locations where the go server is looking for the authorities in https://go.dev/src/crypto/x509/root_linux.go

**Reply by svicknesh** [reply]
Thank you @ganigeorgiev , I'll give both the methods a try. Thanks for suggesting method (1), since I have my custom docker image, I can always load my CA cert as part of the image. 

Also, thank you for creating such a wonderful tool. It's really helpful for speeding up development, and more lightweight than some other alternatives. Cheers!

**Reply by svicknesh** [reply]
@ganigeorgiev I can confirm method 1 works. Pocketbase can connect to the Minio instance. Haven't tried method 2 yet. When I do, I'll post my feedback.

Thanks for the assist.
